platforms/quorum/configuration/roles/setup/bootnode/tasks/main.yaml

```diff
@@ -42,8 +35,8 @@
 # This task puts the bootnode binary to the bin directory created in the above task
 - name: Copy bootnode binary to destination directory
   copy:
-    src: "{{ bin_install_dir }}/quorum/build/bin/bootnode"
-    dest: "{{ bin_install_dir }}/bootnode"
-    mode: 0755
+    src: "{{ bin_install_dir }}/bootnode"
+    dest: "{{ bin_install_dir }}"
+    mode: 0777
```

Comment:
Is it not working with 755? (Basically its good to give as minimal permissions as possible)

Reply:
done